gui/src/main/java/io/bitsquare/gui/util/MakeSound.java

```diff
@@ -0,0 +1,75 @@
+package io.bitsquare.gui.util;
```

Comment:
since there's nothing gui specific here maybe this should be added to the common module? It's not unthinkable that a headless user might appreciate a beep in certain situations?